apps/docs/src/app/core/component-docs/illustrated-message/examples/illustrated-message-spot-example.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-illustrated-message-spot-example',
+    templateUrl: './illustrated-message-spot-example.component.html'
```

Comment:
don't we need `changeDetection: ChangeDetectionStrategy.OnPush` for the examples?

Reply:
But why do we need OnPush strategy in the documentation?

Reply:
as discussed, not needed here. Thanks for clarifying!